src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumbsSearch.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import TextField from 'app/components/forms/textField';
+import {IconSearch} from 'app/icons/iconSearch';
+import {t} from 'app/locale';
+import {IconClose} from 'app/icons/iconClose';
+import space from 'app/styles/space';
+
+type Props = {
+  searchTerm: string;
+  onChangeSearchTerm: TextField['props']['onChange'];
+  onClearSearchTerm: () => void;
+};
+
+const BreadCrumbsSearch = ({
+  searchTerm,
+  onChangeSearchTerm,
+  onClearSearchTerm,
+}: Props) => (
+  <Wrapper>
+    <StyledTextField
+      name="breadcumber-search"
+      placeholder={t('Search breadcrumbs...')}
+      autoComplete="off"
+      value={searchTerm}
+      onChange={onChangeSearchTerm}
+    />
+    <StyledIconSearch />
+    <StyledIconClose show={!!searchTerm} onClick={onClearSearchTerm} circle />
+  </Wrapper>
+);
+
+export default BreadCrumbsSearch;
+
+const Wrapper = styled('div')`
+  position: relative;
+  display: flex;
+  align-items: center;
+`;
+
+const StyledTextField = styled(TextField)<TextField['props']>`
+  margin-bottom: 0;
+  input {
+    padding-left: ${space(4)};
+    padding-right: ${space(4)};
+    height: 28px;
+  }
+`;
+
+const StyledIconSearch = styled(IconSearch)`
+  position: absolute;
```

Comment:
Do these need to be absolutely positioned?

Reply:
yes :) but I am going to refactor this filter very soon